### PR TITLE
Update docker image

### PR DESCRIPTION
**Pull request by ChristianLieven**
Modify the tags to manually force an update of the running image via new travis builds. This is handy for instance when dependencies change, but not the package itself.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,8 +16,8 @@ env:
 before_install:
 - echo ${DOCKER_PASSWORD} | docker login -u=decaftravis --password-stdin
```
Comment (Midnighter) [inline .travis.yml]: Untested but something like the following should be added in the `before_install` section:
```yaml
  - docker build -t ${IMAGE_REPO}:update .
  - docker run --rm --volume ".:/home/kaa/app" --user "kaa" ${IMAGE_REPO}:update pipenv lock
```

Comment (Midnighter) [reply .travis.yml]: I think the `Pipfile.lock` should then be removed from the repo.

Comment (Midnighter) [reply .travis.yml]: My bad, `docker run --rm --volume ".:/home/kaa/app" ...` only works in docker-compose. It should be `docker run --rm --volume "${PWD}:/home/kaa/app" ...`

**Comment by kvikshaug** [comment]
This sounds like good idea but I'm closing it as the PR has grown stale, for example we're not using pipenv anymore. Feel free to submit an updated version though.

**Pull request closed without merging**